addons/point_of_sale/static/src/xml/Screens/PaymentScreen/PaymentScreen.xml

```diff
@@ -13,7 +13,7 @@
                         </div>
                         <div class="top-content-center"><h1>Payment</h1></div>
                         <div class="button next" t-att-class="{ highlight: currentOrder.is_paid() and currentOrder._isValidEmptyOrder() }"
-                            t-on-click="trigger('validate-order')">
+                              t-on-click="lockedValidateOrder(false)">
```

Comment:
```suggestion
                              t-on-click="() => lockedValidateOrder(false)">
```

Make this a lazy so that when it reached 15.3, it is already compatible with owl2. Do the same on the other t-on-click.